tsconfig.build.json

```diff
@@ -1,9 +1,8 @@
 {
   "extends": "./tsconfig.json",
+  "include": ["./lib"],
```

Comment:
I've moved this from the `tsconfig.cjs.json` and `tsconfig.esm.json`, since both of them share the same configs, and this field was the same between them.